modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/languages/RetrofitClientCodegen.java

```diff
@@ -0,0 +1,190 @@
+package com.wordnik.swagger.codegen.languages;
+
+import com.wordnik.swagger.codegen.*;
+import com.wordnik.swagger.models.Operation;
+import com.wordnik.swagger.models.properties.*;
+
+import java.util.*;
+import java.io.File;
+
+public class RetrofitClientCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Yes, so it'll be better to wait for changes from #805 to be applied (or not), then to make a decision about generalisation java clients